keen/explorer
# Fixes #155

## What does this PR do?
It fixes #55.

## How can I test this?
1) Create a simple count query.
2) Add a filter with the `contains` operator and a String property value that's made of only numeric characters, like "1".
3) Run the query and ensure you get at least 1 result. If you don't - change the event collection or change the property value number to something else.
4) Save the query.
5) Reload the page. Run the query again. Ensure you get the same result.

## Does this have tests?
It sure does!
https://github.com/keen/explorer/compare/FIX-155?expand=1#diff-0f00529d4845b3fefd78d8fa03ecbc36R16
